string map_name
---
bool result
